exo-go/features/remove_template/remove_template.feature

```diff
@@ -0,0 +1,35 @@
+Feature: removing existing templates
+
+  When developing an exosphere application
+	I want to be able to remove existing service templates from my application codebase
+	So that I don't have to manually run the git command to remove the submodules
+
+  - run "exo remove-template" to remove an existing service template from the application codebase
+
```

Comment:
We are no longer storing the template info in `application.yml` so I guess this is irrelevant.